style.css

```diff
@@ -86,3 +92,179 @@ h1 span {
 .Menu {
   padding-right: 30px;
 }
+
+h2 {
+  margin-bottom: 20px;
+  width: 327px;
+  height: 97px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.Works {
+  margin-top: 62px;
+  width: 375px;
+  display: grid;
+  row-gap: 24px;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto auto auto auto;
+  justify-content: center;
+  align-items: center;
```

Comment:
- [ ] I think it would be better to use `width:100%` or `width:100vw` for the main container to make the website responsive, this goes also for the previous section and header. :recycle:

Reply:
Done. Thanks for your feedback. @GedeonTS